.travis.yml

```diff
@@ -1,4 +1,6 @@
-sudo: false
+sudo: true
```

Comment:
What is it about this set of changes that required us to flag for sudo? Was it the use of trusty here?

Reply:
Leftover. The xenial Travis image was only available with sudo for a while.

Reply:
I think there might be faster perf on boot up for non-sudo images. Since we don't need anything from sudo atm, I was wondering if we should keep this change?